arrows/ocv/match_features_bruteforce.cxx

```diff
@@ -43,10 +43,10 @@ namespace ocv {
 class match_features_bruteforce::priv
 {
 public:
-  priv(int norm_type=cv::NORM_L2, bool cross_check=false)
-      : norm_type( norm_type ),
-        cross_check( cross_check ),
-        matcher( new cv::BFMatcher(norm_type, cross_check) )
+  priv(int n_type = cv::NORM_L2, bool c_check = false)
```

Comment:
Yeah, this is one of those times I'd tell `-Wshadow` to go get bent... not saying I'm requesting that, just saying that's what I'd do.